### ISSUE TITLE
Adjust Font Color to Slightly Intense (Not Pure White/Black)

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#387]
# Description
```
Please provide a short description of what you have encountered below.
```
As Sibert feedback and test on my field testing, it's required to adjust the font color with slightly more intensity but not pure white/black. The current grey text is very annoying after long read via high-res screen like Apple Retina.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
Adjust black to: `var(--color-typography-900)`
Adjust white to: `var(--color-typography-50)`

# Current Behavior
```
Please specify the current behavior of software below.
```
They're grey (`var(--color-typography-700)`).

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```
Write something very long (like copy pasting a long paragraph) using `zoralabCORE` UI component on high contrast design (e.g. black on white and inverted). Observe the annoyance.

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 5096a8bb838553ab34e94ec21b6916f07628bdd9

**Comment by hollowaykeanho** [comment]
Released in v1.2.1.
